.buildkite/scripts/steps/functional/security_serverless.sh

```diff
@@ -10,4 +10,4 @@ export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
 
 echo "--- Security Serverless Cypress"
 
-yarn --cwd x-pack/test_serverless/functional/test_suites/security/cypress cypress:run
+yarn --cwd x-pack/plugins/security_solution cypress:run:serverless
```

Comment:
So this line replaces this one:

```bash
yarn --cwd x-pack/test_serverless/functional/test_suites/security/cypress cypress:run
```

But in `x-pack/test_serverless/functional/test_suites/security/cypress` folder there are some tests already that EDR Workflows team started to add there. Have you discussed it with them?

cc @kevinlog @paul-tavares @patrykkopycinski

Reply:
Thanks for the ping @banderror .
Until we can move to whatever the end setup is going to be, I would suggest that we continue to run the existing `x-pack/test_serverless/functional/test_suites/security/cypress` test, which are currently validating serverless specific logic (that does not exist in non-serverless).

Reply:
> But in x-pack/test_serverless/functional/test_suites/security/cypress folder there are some tests already that EDR Workflows team started to add there. Have you discussed it with them?

I've shared the PR in different channels including Serverless specific ones. 

Good catch!! We'll keep that execution as well :)